e2e/utils/ScreenCatcher.ts

```diff
@@ -31,11 +32,21 @@ export class ScreenCatcher {
         }
 
         const date: Date = new Date();
-        const timeStr: string = date.toLocaleTimeString('en-us', {hour12: false}) + '.' + new Intl.NumberFormat('en-us', {minimumIntegerDigits: 3}).format(date.getMilliseconds());
+        const timeStr: string = date.toLocaleTimeString('en-us', { hour12: false }) + '.' + new Intl.NumberFormat('en-us', { minimumIntegerDigits: 3 }).format(date.getMilliseconds());
 
         const screenshotPath: string = `${executionScreenCastDir}/${formattedMethodIndex}${formattedScreenshotIndex}--(${timeStr}): ${methodName}.png`;
 
-        await this.catchScreen(screenshotPath);
+        try {
+            await this.catchScreen(screenshotPath);
+        } catch (err) {
+            if (!fs.existsSync(executionScreenCastErrorsDir)) {
+                fs.mkdirSync(executionScreenCastErrorsDir);
+            }
+
+            let errorLogFilePath: string = screenshotPath.replace('.png', '.txt');
```

Comment:
Here we can just add `.txt`, or, more obvious, `.error.log` to the **screenshotPath**, to have file name like the follow:
> 0010002--(10:04:38.853): Open workspace.png.error.log